<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QMainWindow" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>485</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>485</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icons/icon/Chat-icon.png</normaloff>:/icons/icon/Chat-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="receiveTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>271</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="YouToolButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icons/icon/icq_online.jpg</normaloff>:/icons/icon/icq_online.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="myToolButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icons/icon/icq_online.jpg</normaloff>:/icons/icon/icq_online.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>81</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="qqLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>158</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>关闭对话</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendBtn">
       <property name="text">
        <string>发送信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>341</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="toolButton_Font">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/qq/icon/qq/aio_favourite_cell_normal@2x.png</normaloff>:/qq/icon/qq/aio_favourite_cell_normal@2x.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Palette">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icons/icon/textcolor.png</normaloff>:/icons/icon/textcolor.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Bold">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icons/icon/textbold.png</normaloff>:/icons/icon/textbold.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Italic">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icons/icon/textitalic.png</normaloff>:/icons/icon/textitalic.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Underline">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icons/icon/textunder.png</normaloff>:/icons/icon/textunder.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_ScreenShot">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/QQ/icon/qq/toolbar_screenCapture_hover@2x.png</normaloff>:/QQ/icon/qq/toolbar_screenCapture_hover@2x.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Nudge">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/QQ/icon/qq/toolbar_shake_hover@2x.png</normaloff>:/QQ/icon/qq/toolbar_shake_hover@2x.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_SendFile">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/QQ/icon/qq/titlebar_function_fileTransfer_hover@2x.png</normaloff>:/QQ/icon/qq/titlebar_function_fileTransfer_hover@2x.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Save">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/QQ/icon/qq/toolbar_history_hover@2x.png</normaloff>:/QQ/icon/qq/toolbar_history_hover@2x.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Send_File">
   <property name="text">
    <string>&amp;Send File</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>sendBtn</tabstop>
  <tabstop>sendTextEdit</tabstop>
  <tabstop>toolButton_Font</tabstop>
  <tabstop>toolButton_Palette</tabstop>
  <tabstop>toolButton_Bold</tabstop>
  <tabstop>toolButton_Italic</tabstop>
  <tabstop>toolButton_Underline</tabstop>
  <tabstop>toolButton_SendFile</tabstop>
  <tabstop>YouToolButton</tabstop>
  <tabstop>myToolButton</tabstop>
  <tabstop>receiveTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>Chat</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>Chat</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
